src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/jobs/PromptLicenceCreationController.kt

```diff
@@ -55,7 +59,44 @@ class PromptComController(
       ),
     ],
   )
-  fun run(): List<Com> {
-    return promptComService.runJob()
+  fun run(): List<PromptComNotification> {
+    return promptComService.getCases()
+  }
+
+  @ProtectedByIngress
```

Comment:
indicates to the role integration test that this endpoint is protected by the ingress so doesn't need role based security.